gatorgrade/output/output_functions.py

```diff
@@ -1,7 +1,57 @@
-"""Add docstring."""
+"""
+This class is used for storing the main functions requested from the Github.
 
+Issue Tracker for the output team.
+For instance, functions dealing with percentage output, description output,
+and colorization of text.
+"""
+import colorama as color
 
-# This class is used for storing the main functions requested
-# from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output,
-# description output, and colorization of text.
+color.init()
+
+
+def sort_checks_by_result(results):
+    """
+    Process results and determine if the check passed or failed.
+
+    Argument `results` is: list[(check_result_element1, check_result_element2),(...)]]
+    """
+    passed_checks = []
+    failed_checks = []
+    # iterate through results tuples
+    for i in results:
+        for j in i:
+            if isinstance(j, bool):
+                if j is True:
+                    passed_checks.append(i)
+                else:
+                    failed_checks.append(i)
+    output_passed_checks(passed_checks)
+    output_failed_checks(failed_checks)
+
+
+def output_passed_checks(passed_checks):
+    """Output the results for all of the checks that passed using the passed_checks list."""
+    for i in passed_checks:

```

Comment:
I would recommend renaming `i` to a more descriptive variable name.

Reply:
Great idea, I changed it to `check` in my latest push!